packages/doc/docusaurus.config.js

```diff
@@ -64,6 +64,11 @@ const config = {
             position: 'left',
             label: 'Document',
           },
+          {
+            to: '/docs/contributing/overview',
+            position: 'left',
+            label: 'Contributing',
+          },
```

Comment:
Since we haven't had the document about contributing, can we remove this item in the PR?

Reply:
Thanks for the suggestion, I have removed it.